aries-site/src/pages/components/tabs.mdx

```diff
@@ -33,6 +36,14 @@ the browser location.** This will allow proper active styling and routing to fun
 
 **If using Tabs as a navigation structure, don't use them in conjunction with other navigation structures.** Combining multiple navigation structures creates a confusing experience for the user. It can lead to difficulties in navigating through an application and in ability to remember where desired content lives.
 
+### Control Alignment
+
+Apply `justify="start"` to the Tabs so the tab controls align to the start of their container and the bottom border extends the full-width of the parent container.
```

Comment:
Suggestion: 

I feel like I want all the alignment content under a single heading, starting with a purpose statement. For example:

"To maintain a clean, consistent presentation, all content should be aligned to the left edge of the Tabs component.
- Apply `<Tabs justify="start" ... />` to left-align the tabs and have their bottom border extend the full-width of the parent container.
- Content within the Tab should be flush with the left edge of the Tabs component, and have a vertical padding of `medium` to provide spacing and room for the content to breathe."

Soft opinion. Also I was getting thrown off by "tab controls" -- was causing me doubt as to are those the tabs? Someother type of control? etc? Was seeing if this might help with that as well.

Reply:
That's a fair comment. Let me consolidate.

Reply:
Done.